python/examples/dataframe/concat.py

```diff
@@ -19,6 +17,13 @@
 # distributed unique
 env = CylonEnv(config=MPIConfig())
 
-print("Distributed Unique", env.rank)
-df4 = df1.concat(axis=0, objs=[df2, df3], env=env)
-print(df4)
+if env.world_size > 1:
```

Comment:
do we need this check here?

Reply:
Not required. But it would be meaningless to say it as distributed otherwise. This can be removed, if it is not necessary.

Reply:
My suggestion is to remove it. Having an unwanted line can complicate things